pkg/deploy/controller/deployment/controller.go

```diff
@@ -249,6 +250,10 @@ func (c *DeploymentController) makeDeployerPod(deployment *kapi.ReplicationContr
 		},
 	}
 
+	// MergeInfo will not overwrite values unless the flag OverwriteExistingDstKey is set.
+	util.MergeInto(pod.Labels, deploymentConfig.Template.Strategy.Labels, 0)
```

Comment:
Please suggest a name for the constant. There are several  flags that are supported and it is not obvious what should be used for the absence of any flags.